README.md

```diff
@@ -0,0 +1,228 @@
+<a name="My Portfolio"></a>
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [your_project_name] <a name="My Portfolio">My Portfolio</a>
+
+> Describe your project in 1 or 2 sentences.
+This is a My Portfolio  project.
+**[your_project__name]** is My Portfolio
+
+## 🛠 Built With <a name="built-with">Html, css</a>
+
+### Tech Stack <a name="tech-stack">HTML5, CSS3</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+HTML5 and css3 which are the latest version were used in this project.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]** My Portfolio built with HTML and CSS
+- **[key_feature_2]** Responsive mobile design portfolio
+- **[key_feature_3]**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project. Yet to be deployed.
+
+- [Live Demo Link]
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+ Get your pc.
+ Clone this project to your local machine. 
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+Basic knownledge of computer.
+
+In order to run this project you need:
+live server.
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/EnixWord/MY_PORTFOLIO.git
+
+### Install
+
+Install this project with:
+cd to My Portfolio.
+
+### Usage
+
+To run the project, execute the following command:
+live server
+<!--
+Example command:
+
+```sh
+  
+```
+--->
+
```

Comment:
- Please, let's remove this commented code here also as they're redundant. You can choose to add something different related to Usage, or remove the `Usage` section.